packages/ketcher-core/src/application/formatters/formatProperties.ts

```diff
@@ -98,6 +98,12 @@ const formatProperties: FormatPropertiesMap = {
   cdx: new SupportedFormatProperties(
     'Base64 CDX',
     ChemicalMimeType.CDX,
+    ['.b64'],
```

Comment:
As I understand `.cdx` is a binary file format ([Wiki](https://en.wikipedia.org/wiki/CDX_Format)) while we are talking about `base64` file format. If we change the extension it will result in an error as both ketcher and marvin expect a binary file format

Reply:
Changed to .b64cdx